datascience/tables.py

```diff
@@ -5719,12 +5719,48 @@ def asdict(self):
             return collections.OrderedDict(zip(self._table.labels, self))
 
     class Rows(collections.abc.Sequence):
-        """An iterable view over the rows in a table."""
+        """
+        An iterable row-styled view of the table object that is passed as an argument.
```

Comment:
Let's keep the original statement here - the intended/standard usage of this class doesn't have users manually creating this class.